grapher/lineCharts/LineChartUtils.ts

```diff
@@ -10,12 +10,20 @@ export const columnToLineChartSeriesArray = (
     const { isProjection, owidRowsByEntityName } = col
     const entityNames = Array.from(owidRowsByEntityName.keys())
     return entityNames.map((entityName) => {
-        const seriesName =
-            seriesStrategy === SeriesStrategy.entity
-                ? entityName
-                : canSelectMultipleEntities
-                ? `${entityName} - ${col.displayName}`
-                : col.displayName
+        let seriesName = ""
+        if (seriesStrategy === SeriesStrategy.entity) {
+            if (col.isProjection && col.displayName) {
+                seriesName = `${entityName} - ${col.displayName}`
```

Comment:
Do you have an example chart where this applies? Can we get away with not appending `${col.displayName}`?

Reply:
I found these: http://localhost:3030/admin/test/embeds?ids=682,688,958,1006,1133,1193,1194,1480,2822,3265

Did you make this change with the intention of matching previous behaviour (currently on live)?

I think it might be better not to keep the "- Projection" suffix when `SeriesStrategy.entity`? (I think this is how it was before Next)